var/spack/repos/builtin/packages/openpmd-api/package.py

```diff
@@ -50,7 +50,7 @@ class OpenpmdApi(CMakePackage):
 
     depends_on('cmake@3.10.0:', type='build')
     depends_on('mpark-variant@1.3.0:')
-    depends_on('catch@2.2.1: ~single_header', type='test')
+    depends_on('catch@2.3.0: ~single_header', type='test')
```

Comment:
@healther `openmd` seems like @ax3l 's project, so I am pretty confident this works :smile: I'll merge this, let me know if further corrections are needed.

Reply:
My only version in spack is our github HEAD so far :)
(Since I want to avoid maintaining all my initial design errors in spack mainline during alpha state of the software.)

Reply:
Thanks for the explanation :)

Reply:
Thank you for the review :)